docs/components_page/components/tabs/simple.py

```diff
@@ -0,0 +1,35 @@
+import dash_bootstrap_components as dbc
+import dash_html_components as html
+
+tab1_content = dbc.Card(
+    dbc.CardBody(
+        [
+            dbc.CardText("This is tab 1!"),
+            dbc.Button("Click here", color="success"),
+        ]
+    ),
+    # add small margin to the top of the card
+    className="mt-3",
+)
+
+tab2_content = dbc.Card(
+    dbc.CardBody(
+        [
+            dbc.CardText("This is tab 2!"),
+            dbc.Button("Don't click here", color="danger"),
+        ]
+    ),
+    # add small margin to the top of the card
```

Comment:
I'm not sure this comment (and the above) add that much value. At least, we should be able to have just one.